packages/notifier/README.md

```diff
@@ -166,74 +173,70 @@ const {
-`AsyncIterable` API. Since each published value will be sent to all
-subscribers, the SubscriptionKit should generally not be used with rapidly
-produced values.
-
-The values published using the publication define the original iteration. Each
-subscription has a starting point in that iteration and provides access to a
-suffix subset of that iteration starting at that starting point. The initial
-subscription created by the `makeSubscriptionKit()` call provides the entire
-iteration. Each subscription is a kind of `AsyncIterable` which produces any
-number of `AsyncIterators`, each of which advance independently starting with
-that subscription's starting point. These `AsyncIterators` are
-`SubsciptionIterators` which also have a `subscribe()` method. Calling a
-SubscriptionIterator's `subscribe()` method makes a `Subscription` whose
-starting point is that `SubscriptionIterator`'s current position at that time.
-
-Neither Alice nor Bob are good starting points to construct an example of
-`subscribe()` since their code uses only a `Subscription`, not
-a `SubscriptionIterator`. Carol's code is like Bob's except lower level, using
-a `SubscriptionIterator` directly. Where Bob uses `observeIteration` which
-takes an AsyncIterable`, Carol's uses the lower level `observeIterator` which
-takes an `AsyncIterator`.
+An iteration’s *sampling subset*:
+   * May omit some of the original iteration’s non-final values. 
+   * All sampling subset non-final values are in the original’s non-final values in the same order. 
+   * The original and the subset both have the same termination. 
+   * Once an original iteration value is available, either that value or a later one will 
+      become available on each sampling subset *promptly*, i.e. eventually and without waiting   
+      on any other manual steps. In other words, If a value 'a' is introduced on the producer end,  
+      then all clients either promptly see 'a', or won't see 'a', but will promptly see a successor. So 
+      if two values are added in quick succession, the first might not be visible to all consumers. 
+      But if a value is added and nothing follows quickly, then that value must be distributed 
+      promptly to the consumers.
+
+If the most common, or your expected, consumer, only cares about the current state, then use a Notifier. 
+If some important consumers need to see all the values, a Subscription serves their needs. This is often
+appropriate when the iteration represents a changing quantity, like a purse balance, and a consumer 
+updating a UI that doesn't care to hear about any older non-final values, as they are more stale. A 
+Notifier is appropriate even when this quantity changes quickly, as it only communicates non-final values
+at the rate they're being consumed, bounded by the network round-trip time. All other non-final values 
+are never communicated. The Notifier's lossy nature enables this optimization.
```

Comment:
```suggestion
are never communicated. The NotifierKit's lossy nature enables this optimization.
```
The thing is the NotifierKit. The Notifier and the Updater are the two sides of the NotifierKit. Likewise, SubscriptionKit rather than Subscription.